Presentation.Web/Tests/PageObjects/it-system/tabs/ItSystemReference.po.ts

```diff
@@ -0,0 +1,18 @@
+﻿import IPageObject = require("../../IPageObject.po");
+import KendoToolbarWrapper = require("../../../object-wrappers/KendoToolbarWrapper")
+
+var ec = protractor.ExpectedConditions;
+
+class ItSystemEditPo implements IPageObject { 
```

Comment:
Det er vel ItSystemEditReferencePo?

Reply:
og filnavnet skal vel også passe

Reply:
Og det er kun for "DefaultItSystem".. måske skulle getPage tage et argument, så man kan vælge på hvilket system den skulle åbne